org.freedesktop.Sdk.Extension.gnat14.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Junde Yhi <junde@yhi.moe> -->
+<component type="addon">
+  <id>org.freedesktop.Sdk.Extension.gnat14</id>
+  <extends>org.freedesktop.Sdk</extends>
+  <name>GNAT 14</name>
+  <summary>GNU Ada/SPARK compiler 14.x and build environment</summary>
+  <description>
+    <p>This is a Freedesktop SDK extension containing necessary components to build an Ada/SPARK project:</p>
+    <ul>
+      <li>GNAT - the GNU Ada compiler</li>
+      <li>GNATprove - the GNU SPARK analyzer</li>
+      <li>GPRbuild - the GNAT Project Manager, with companion tools</li>
+      <li>Alire (alr) - the Ada Library Repository tool</li>
+    </ul>
+  </description>
+  <categories>
+    <category>Development</category>
+  </categories>
+  <developer id="dev.ada.alire">
+    <name>Ada Library Repository</name>
+  </developer>
+  <content_rating type="oars-1.0" />
```

Comment:
```suggestion
  <content_rating type="oars-1.1" />
```

Reply:
<https://hughsie.github.io/oars/generate.html> says 1.1 works with GNOME Software >= 3.27.3. Won't a 1.0 markup here provide more compatibility (provided that this is an empty tag)?

Reply:
No, that website hasn't been updated it years and the version is ancient.

Reply:
Changed.